TACC/tup-ui
# Add plugin for rapid generation of staff pages - split name into first and last

## Overview & Changes

Split name into first name and last name.

<sup>The other fields will be added in #269. I made this its own PR, because I am altering an existing field with data, which  is new to me.</sup>

## Related

- changes https://github.com/TACC/tup-ui/pull/253

## Testing

0. Run local environment.
1. `docker exec -it tup_cms /bin/bash`
2. `python manage.py migrate`
3. Open CMS (be logged in).
4. Add Staff Profile plugin.
5. Verify First Name and Last Name fields exist.
6. Verify First Name and Last Name fields are required.

## UI

![first and last name](https://github.com/TACC/tup-ui/assets/62723358/65bc33e3-7691-4013-b1f6-46d5c8594b95)
